test/integration/auth-methods/ntlm.test.js

```diff
@@ -136,13 +78,13 @@ describe('NTLM', function () {
                     value: 'postman'
                 }, {
                     key: 'pass',
-                    value: 'password'
```

Comment:
Can you move username and password to the top of the file?

Reply:
sure 👍